data-visualize.qmd

```diff
@@ -156,33 +150,25 @@ ggplot(mpg,aes(x = hwy, y = displ,size = cty)) + geom_point()
 ggplot(mpg,aes(x = hwy, y = displ,size = cty,color = cty)) + geom_point()
 ```
 
-3.In the scatterplot of `hwy` vs. `displ`, what happens if you map a third variable to `linewidth`?
-
-There is no line to alter.
+3. There is no line to alter.
 
 ```{r}
 ggplot(mpg,aes(x = hwy, y = displ,linewidth = cty)) + geom_point()
 ```
 
-4.what happens if you map the same variable to multiple aesthetics?
-
-This code only shows 1 variable compared to itself,which isn't helpful.
+4. This code only shows 1 variable compared to itself,which isn't helpful.
 
 ```{r}
 ggplot(mpg,aes(x = hwy, y = hwy, color  = hwy)) + geom_point()
 ```
 
-5.Make a scatterplot of `bill_depth_mm` vs. `bill_length_mm` and color the points by `species`. What does adding coloring by species reveal about the relationship between these two variables?
-
-That adelies tend to have more depth in their bills while gentoo are longer all while chinstrap are both long and have depth.
+5. That adelies tend to have more depth in their bills while gentoo are longer all while chinstrap are both long and have depth.
 
 ```{r}
 ggplot(penguins,aes(x = bill_depth_mm, y = bill_length_mm,color = species)) + geom_point()
 ```
 
-6.Why does the following yield two separate legends? How would you fix it to combine the two legends?
-
-Because the labs argument makes another legend, i just took that argument out.
+6. Because the labs argument makes another legend, i just took that argument out.
```

Comment:
```suggestion
6. The code provided in the exercise yields two separate legends because the legend for `color` is renamed to `"Species"` but the legend for shape is not, and is named `"species"` by default instead. To fix it, we would need to explicitly rename the shape legend as well.
```